issue_template.md

```diff
@@ -0,0 +1,14 @@
+# Configuration
+
+Run `react-native info` in your project and share the content.
+What `react-native-track-player` version are you using?
+
+# Issue
+/*
+  Please, describe with details the problem, let us know which platform (Android, iOS or Windows) and which OS version are you experiencing this issue.
+  Crash logs, step-by-step instructions, code snippets and device details may also help us identify and reproduce the issue.
+*/
```

Comment:
Put it between `<!-- -->` instead, since GitHub recognizes it as an actual comment

Reply:
done (I thought it wasn't visible but it is indeed)